js/_actions.js

```diff
@@ -0,0 +1,147 @@
+// polyfills
+require('babel-polyfill');
+var fetch = require('isomorphic-fetch');
+
+export const SET_MESSAGE = "SET_MESSAGE";
+export function setError(errorMessage) {
+	return {
+		type: SET_MESSAGE,
+		message: errorMessage,
+		message_type: "error"
+	};
+}
+
+export const SET_API_ENDPOINT = "SET_API_ENDPOINT";
+export function setAPIEndpoint(url) {
+	return {type: SET_API_ENDPOINT, url: url};
+}
+
+export const SET_ACTIVE_STEP = "SET_ACTIVE_STEP";
+export function setActiveStep(id) {
+	return {type: SET_ACTIVE_STEP, id};
+}
+
+export const GIT_REQUEST_UPDATE = 'GIT_REQUEST_UPDATE';
```

Comment:
Or sort of following the same standard as [redux-saga](https://github.com/yelouafi/redux-saga/#sagasjs)

Name them in the past sense, 
```
USER_FETCH_REQUESTED
USER_FETCH_SUCCEEDED
USER_FETCH_FAILED
```